opencontrail/map.jinja

```diff
@@ -79,7 +84,7 @@ database:
 web:
   Debian:
     pkgs:
-         ['contrail-web-controller', 'nodejs-legacy', 'gettext-base']
+         ['contrail-web-controller', 'nodejs', 'gettext-base']
```

Comment:
pls, provide reasonings why nodejs-legacy was in conflict, this is not expected to be needed.
I quite guess we have on on container nodejs installed by some other dependency that in the end conflicts to nodejs-legacy.